### ISSUE TITLE
Should return generic response on password reset request

### ISSUE THREAD

**Issue by rpunnett** [TryGhost/Ghost#8529]
### Issue Summary

When using the "Forgot Password" feature on the URI 'ghost/signin' page, a non-generic message is returned that the email address does not exist. The reverse is also true for an address that does exist.

Fail Message: "There is no user with that email address."
Success Message: "Please check your email for instructions."

### Steps to Reproduce

1. Access the sign in page: 'blog_url.com/ghost/signin/'
2. In the username/email input, enter in an email address that does not exist
3. Select the "Forgot?" option from the password input
4. A message will be displayed that the user does not exist

This action is contrary to security best practices. Only generic messages should be returned. An example would be: "If a user account exists with this email address, an email has been sent with instructions to reset your password". Returning descriptive error messages can allow for brute forcing of accounts.

### Technical details:

* Ghost Version: 0.11.19
* Node Version: N/A
* Browser/OS: N/A
* Database: N/A

**Comment by kevinansfield** [comment]
Hey @rpunnett 👋

This issue has been discussed previously in the [original feature issue](https://github.com/TryGhost/Ghost/issues/468) and [at least once since](https://github.com/TryGhost/Ghost/issues/1453).

TL;DR: We have brute force protection in place that mitigates much of the email enumeration risk so we're happy with the tradeoff in keeping the more user-friendly messages.

For further security issues, it would be great if you could follow https://github.com/TryGhost/Ghost/blob/master/SECURITY.md. Thanks 😄
